src/routes/index.js

```diff
@@ -111,6 +111,7 @@ export default function getRouteHandlers(
     'POST /tools/import': importController.createImportJob,
     'GET /tools/import/:jobId': importController.getImportJobStatus,
     'GET /tools/import/:jobId/result': importController.getImportJobResult,
+    'GET /tools/import/jobs': importController.getImportJobsByDateRange,
```

Comment:
Thinking out loud, if we want to enable the client to retrieve all their jobs (for a specific API key), would it be this same endpoint? How would the service distinguish between a request for all jobs, vs. only jobs for key "X"?

I wonder if a `GET /tools/import/jobs/by-date-range` endpoint for this use case may work well instead?

Reply:
is the `api-key` not part of the "auth context" and would intrinsically be applied to a query if so needed? i don't think it has to leak into http api unless a user (same auth context) can search for multiple different keys.

i support the idea to have specific endpoints like `by-date-range` - otherwise we'll be forced to create a super generic query interface, which at some point may be needed but not now.

Reply:
For our usecase, we want to retrieve all jobs (regardless of the jobs tied to a specific API key). Also, we don't want to let customers access the import jobs of other customers. 
For now, I'll go ahead with `GET /tools/import/jobs/by-date-range` endpoint.